pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -143,7 +143,7 @@ func envoyMetricsFilter(conf api.Conf) string {
 	if query != "" {
 		query += "&"
 	}
-	if pointer.Deref(conf.Sidecar.UsedOnly) {
```

Comment:
i think that this won't work by default because of check: in line `136`

Reply:
modified, can you check again?